prosody/rootfs/etc/cont-init.d/10-config

```diff
@@ -1,5 +1,42 @@
 #!/usr/bin/with-contenv bash
 
+if [[ "$ENABLE_LDAP_AUTH" == "yes" || "$ENABLE_LDAP_AUTH" == "1" || "$ENABLE_LDAP_AUTH" != "true" ]]; then
+
+  if [[ ! -d /var/run/saslauthd ]]; then
+      mkdir -p /var/run/saslauthd
+      chmod 777 /var/run/saslauthd
+  fi
+
+  [[ "${LDAP_URL}" ]] && echo >> /etc/saslauthd.conf "ldap_servers: ${LDAP_URL}"
```

Comment:
doesn't it make sense to ship a templated file instead of all there echoes? which BTW will flood the file on every boot AFAIS